samples/snippets/src/main/java/com/example/video/DetectPerson.java

```diff
@@ -14,25 +14,25 @@
  * limitations under the License.
  */
 
-package beta.video;
+package com.example.video;
```

Comment:
We should avoid `com.example` except for legacy samples, let's not move these into `com.example`.

https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#package-names